### ISSUE TITLE
I succeed to export onnx of model with Recognizer3D type, but failed from the model with Recognizer2D type, please give some hints, thank you!

### ISSUE THREAD

**Issue by newforrestgump001** [open-mmlab/mmaction2#1638]
for type='Recognizer3D', it works:
python tools/deployment/pytorch2onnx.py configs/recognition/tpn/tpn_imagenet_pretrained_slowonly_r50_8x8x1_150e_kinetics_rgb.py works_dir_tpn_imagenet_pretrained_slowonly_r50_8x8x1_150e_kinetics_rgb/best_top1_acc_epoch_5.pth  --softmax --shape 1 1 3 7 256 256 --verify --show --output-file output.onnx

but for type='Recognizer2D', it failed.
python tools/deployment/pytorch2onnx.py configs/recognition/tanet/tanet_r50_1x1x8_50e_sthv1_rgb.py 
 tanet_best_top1_acc_epoch_3.pth --softmax --shape 3 7 256 256 --verify --show --output-file output.onnx

Please give me some hints and sincere thanks for all of you!

**Comment by hukkai** [comment]
Could you show the error logs?

**Comment by iucario** [comment]
Should be `--shape 1 8 3 256 256`
